pkg/action/executor/resize_container_util.go

```diff
@@ -38,15 +38,15 @@ func updateRequests(container *k8sapi.Container, patchRequests k8sapi.ResourceLi
 	}
 
 	if changed {
-		glog.V(2).Infof("Try to update container %v resource request from %+v to %+v",
-			container.Name, container.Resources.Requests, result)
+		glog.V(2).Infof("Try to update container %v for the workload controller [%v] resource request from %+v to %+v",
+			container.Name, objectID, container.Resources.Requests, result)
```

Comment:
nit: Change "Try to update container %v for the workload controller [%v] resource request from %+v to %+v" to Try to update container %v request from %+v to %v in workload controller [%v]